### DISCUSSION TITLE
[ 已完成 ][ 控件 ] 进度条拖动

### DISCUSSION THREAD

**Discussion by Rainer-Yu** [GeekLee2012/Less-Player-Desktop#14] [Ideas]
因为现在进度提条太窄的原因, 点解切换进度,不好点, 而且没有触发出来拖动进度条的功能

**Comment by GeekLee2012** [comment]
谢谢建议反馈。
已经将ProgressBar，调整为带可拖动滑块的SliderBar。

同时我也尝试优化了一点点用户体验：
将拖动进度对应的时间值，
同步显示到当前歌曲时间进度（即格式为：00:00/00:00，这类控件）。
所以，现在不再是“完全盲狙”，而是“半盲狙”......

代码已同步到Github。
